### ISSUE TITLE
Error: no device attached or simulator installed for device

### ISSUE THREAD

**Issue by Gorniv** [mmcc007/screenshots#73]
after Xcode-beta(11) and return to Xcode(10):
run screenshots
```bash
Starting flutter daemon...
Error: no device attached or simulator installed for device 'iPhone XS Max' in screenshots.yaml.

Guide:
  Attached devices:
Unhandled exception:
ProcessException: No such file or directory
  Command: emulator -list-avds
#0      _ProcessImpl._runAndWait (dart:io-patch/process_patch.dart:489:7)
#1      _runNonInteractiveProcessSync (dart:io-patch/process_patch.dart:634:18)
#2      Process.runSync (dart:io-patch/process_patch.dart:66:12)
#3      cmd (package:screenshots/src/run.dart:375:26)
#4      getAvdNames (package:screenshots/src/utils.dart:159:10)
#5      Config.generateConfigGuide (package:screenshots/src/config.dart:164:31)
#6      Config.validate (package:screenshots/src/config.dart:104:11)
<asynchronous suspension>
#7      run (package:screenshots/src/run.dart:40:16)
<asynchronous suspension>
#8      main (file:///Users/gorniv/github/screenshots/bin/main.dart:79:9)
<asynchronous suspension>
#9      _startIsolate.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:299:32)
#10     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:172:12)
```
```bash
xcrun simctl list devices --json
```
```json
{
  "devices" : {
    "com.apple.CoreSimulator.SimRuntime.iOS-12-0" : [

    ],
    "com.apple.CoreSimulator.SimRuntime.tvOS-12-2" : [

    ],
    "com.apple.CoreSimulator.SimRuntime.iOS-12-2" : [
      {
        "state" : "Shutdown",
        "isAvailable" : true,
        "name" : "iPhone Xs Max",
        "udid" : "3AD11D72-B3FA-4E4C-94B3-E4E51C67250A"
      }
    ],
    "com.apple.CoreSimulator.SimRuntime.watchOS-5-2" : [

    ],
    "com.apple.CoreSimulator.SimRuntime.iOS-9-3" : [

    ],
    "com.apple.CoreSimulator.SimRuntime.iOS-12-1" : [

    ],
    "com.apple.CoreSimulator.SimRuntime.iOS-9-0" : [

    ]
  }
}
```

```
flutter doctor
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, v1.7.8+hotfix.3, on Mac OS X 10.14.5 18F203, locale en-RU)
[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
[✓] Xcode - develop for iOS and macOS (Xcode 10.2.1)
[✓] iOS tools - develop for iOS devices
[✓] Android Studio (version 3.4)
[✓] VS Code (version 1.36.1)
[!] Connected device
    ! No devices available
```

```yaml
# A list of screen capture tests
tests:
  - test_driver/main_store.dart

# Interim location of screenshots from tests
staging: /tmp/screenshots

# A list of locales supported by the app
locales:
  - en-US

# A map of devices to emulate
devices:
  ios:
    iPhone Xs Max:
  # android:
  #   Nexus 6P:

# Frame screenshots
frame: true
```

```
emulator -list-avds
Nexus_6P_API_29
Pixel_API_24
```

**Comment by mmcc007** [comment]
Made a fix for parsing ios simulators.
```
git clone https://github.com/mmcc007/screenshots.git
cd screenshots
git checkout #73_parse_ios_simulators
pub global activate --source path .
```
Let me know if it works.

Not finding emulator command may have been a temporary path issue...

**Comment by Gorniv** [comment]
Thank you, #73_parse_ios_simulators - help me!

**Comment by mmcc007** [comment]
Thanks for the feedback... fix is in release v1.0.2
